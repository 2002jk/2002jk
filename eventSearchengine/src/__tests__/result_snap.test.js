import renderer from 'react-test-renderer';
import SearchResults from "../component/results";

it('Renders search result correctly', () => {
  const tree = renderer
    .create(<SearchResults />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});