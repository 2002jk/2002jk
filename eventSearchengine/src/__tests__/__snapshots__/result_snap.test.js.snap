// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders search result correctly 1`] = `
<div>
  <h1
    className="result-title"
  >
    查询结果
  </h1>
  <button
    className="ant-btn ant-btn-default"
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "marginRight": "10px",
      }
    }
    type="button"
  >
    <span>
      思维导图
    </span>
  </button>
  <button
    className="ant-btn ant-btn-default"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    <span>
      时间线
    </span>
  </button>
  <div
    className="result-tag"
  >
    <a
      href="https://www.baidu.com"
    >
      事件相关信息
    </a>
    <p>
      2022-6-23
    </p>
  </div>
  <div
    className="result-tag"
  >
    <a
      href="https://www.baidu.com"
    >
      事件相关信息
    </a>
    <p>
      2022-6-23
    </p>
  </div>
  <div
    className="result-tag"
  >
    <a
      href="https://www.baidu.com"
    >
      事件相关信息
    </a>
    <p>
      2022-6-23
    </p>
  </div>
</div>
`;
